xbmc/music/ContextMenus.h

```diff
@@ -18,7 +18,7 @@ struct CMusicInfo : CStaticContextMenuAction
 {
   explicit CMusicInfo(MediaType mediaType);
   bool IsVisible(const CFileItem& item) const override;
-  bool Execute(const CFileItemPtr& item) const override;
+  bool Execute(const std::shared_ptr<CFileItem>& item) const override;
```

Comment:
Please add `#include <memory>`to this header.

Reply:
done